internal/common/metrics/constants.go

```diff
@@ -50,12 +50,14 @@ const (
 	ActivitySucceedEndToEndLatency        = TemporalMetricsPrefix + "activity_succeed_endtoend_latency"
 	ActivityTaskErrorCounter              = TemporalMetricsPrefix + "activity_task_error"
 
-	LocalActivityTotalCounter           = TemporalMetricsPrefix + "local_activity_total"
-	LocalActivityCanceledCounter        = TemporalMetricsPrefix + "local_activity_canceled"
-	LocalActivityFailedCounter          = TemporalMetricsPrefix + "local_activity_failed"
-	LocalActivityErrorCounter           = TemporalMetricsPrefix + "local_activity_error"
-	LocalActivityExecutionLatency       = TemporalMetricsPrefix + "local_activity_execution_latency"
-	LocalActivitySucceedEndToEndLatency = TemporalMetricsPrefix + "local_activity_succeed_endtoend_latency"
+	LocalActivityTotalCounter             = TemporalMetricsPrefix + "local_activity_total"
+	LocalActivityCanceledCounter          = TemporalMetricsPrefix + "local_activity_canceled" // Deprecated: Use LocalActivityExecutionCanceledCounter instead.
+	LocalActivityExecutionCanceledCounter = TemporalMetricsPrefix + "local_activity_execution_cancelled"
```

Comment:
Are we using 2 `l`s in Java?

Reply:
Yes: https://github.com/temporalio/sdk-java/blob/c5f0ebbe8a5cd52ee018809ecdd466379b5a7339/temporal-sdk/src/main/java/io/temporal/worker/MetricsType.java

Reply:
yes, go and java match